tests/test_build_recipe.py

```diff
@@ -32,6 +32,7 @@
 LABEL org.opencontainers.image.created="%BUILDTIME%"
 LABEL org.opencontainers.image.vendor="SUSE LLC"
 LABEL org.opencontainers.image.source="%SOURCEURL%"
+LABEL io.artifacthub.package.readme-url="https://raw.githubusercontent.com/SUSE/BCI-dockerfile-generator/sle15-sp4/.readme/test-image/README.md"
```

Comment:
what's the purpose of this extra `/.readme` directory hierarchy? wouldn't it be easier to maintain to store this alongside the Dockerfile in the source directory (so that everything is together)?

Reply:
Including the README in the package directory means that it ends up on OBS/IBS. Currently the README is pretty useless for most packages, hence I thought about not adding it to the main package. But I don't insist on that.